tests/test_graphical_units/test_axes.py

```diff
@@ -14,6 +14,7 @@ def test_axes(scene):
         x_length=6,
         y_length=6,
         color=WHITE,
+        y_axis_config={"tip_shape": StealthTip},
```

Comment:
This seems like a change in defaults if i am not mistaken?

Reply:
I just changed the test case to have an x-axis with the default triangle tip, and the y-axis with the new stealth tip, it shouldn't change any existing default. (Plus, I committed the control data that has been changed as a result of it.)

What do you mean with change in defaults?